truetype.go

```diff
@@ -152,8 +152,10 @@ func NewTruetypeFontConfig(r io.Reader, scale fixed.Int26_6, runeRanges RuneRang
 	return fc, nil
 }
 
-func LoadTruetypeFontConfig(rootPath string) (*FontConfig, error) {
-	fc := &FontConfig{}
+func LoadTruetypeFontConfig(rootPath, name string) (*FontConfig, error) {
+	fc := &FontConfig{} 
+	fc.Name = name;
```

Comment:
Semicolon isn't needed. Running `gofmt` would remove it.

Reply:
Thanks @shurcooL.